tests/e2e/utils/common_utils.go

```diff
@@ -1,16 +1,33 @@
 package utils
 
 import (
+	"encoding/json"
+	"fmt"
 	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
 	"os/exec"
 	"strings"
 )
 
+const (
```

Comment:
create a type and use in `MakeHTTPRequest` function

Reply:
Ah I didn't include that in MakeHTTPRequest, good catch